docs/src/routes/components/components/hero/+page.md

```diff
@@ -1,9 +1,30 @@
 ---
 title: Hero
 breadcrumb: Hero
+nav:
+  - Introductie: "#introduction"
+  - Snelstart: "#quickstart"
+  - Voorbeelden: "#examples"
 ---
 
-<p class="warning">
-  <span>Let op:</span>
-  Deze pagina is nog niet af. De inhoud is nog niet compleet en de voorbeelden zijn nog niet toegevoegd.
-</p>
```

Comment:
Would recommend to still add the warning, since we want to make sure that we don't forget to create the documentation for this component.